docs/src/showcase/missing_physics.md

```diff
@@ -194,9 +194,12 @@ Knowing this, our `predict` function looks like:
 function predict(θ, X = Xₙ[:, 1], T = t)
     _prob = remake(prob_nn, u0 = X, tspan = (T[1], T[end]), p = θ)
     Array(solve(_prob, Vern7(), saveat = T,
-                abstol = 1e-6, reltol = 1e-6))
+                abstol = 1e-6, reltol = 1e-6, 
+                sensealg=ForwardDiffSensitivity()))
```

Comment:
We should probably show `sensealg=QuadratureAdjoint(autojacvec=ReverseDiffVJP(true))`

Reply:
Done!